Resources/Private/CodeTemplates/Extbase/Documentation.tmpl/Developer/Index.rst

```diff
@@ -13,7 +13,7 @@ This is your opportunity to pass on information to other developers who may be u
 
 Use this section to provide examples of code or detail any information that would be deemed relevant to a developer.
 
-You may wish to explain how a certain feature was implemented or detail any changes that might of been
+You may wish to explain how a certain feature was implemented or detail any changes that might of been 
```

Comment:
Trailing whitespace

Reply:
Please fix all these upstream in the the source repo!